### PR TITLE
Hide Building Discovery Image for Satellite

### PR DESCRIPTION
**Pull request by Lennonka**
This was done on 2.5 in #1572 and it must be done for all newer versions as well.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; planned orcharhino 6.4 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: guides/common/modules/proc_implementing-pxe-less-discovery.adoc [hunk 1]
```diff
@@ -28,11 +28,9 @@ ifdef::orcharhino[]
 image::common/pxeless-mode-orcharhino.svg[PXE-less mode]
 endif::[]
 
-ifndef::satellite,orcharhino[]
-image::common/pxeless-mode.svg[]
-endif::[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_implementing-pxe-less-discovery.adoc:33]: Was it intentional to remove the image for Foreman/Katello builds? IMHO these lines should not affect Satellite.

Comment (Lennonka) [reply guides/common/modules/proc_implementing-pxe-less-discovery.adoc:33]: That must a be a mistake, thanks!

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by Lennonka** [comment]
Cherry-picked:
- e4bb1e365..298a284cf  3.7 -> 3.7
- c24db18a1..09dcdd796  3.6 -> 3.6
- dcf0342e8..e627640c5  3.5 -> 3.5
- ad2077860..8e6af302e  3.4 -> 3.4
- 85e3b7186..455285512  3.3 -> 3.3
- 4299c815c..9b2a5fb60  3.2 -> 3.2
- bf103ed8e..7700f476e  3.1 -> 3.1

**Pull request merged**